content/blog/2018/istio-authorization/index.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Micro-Segmentation with Istio Authorization
+description: Describe Istio's authorization feature and how to use it in various use cases.
+publishdate: 2018-07-20
+subtitle:
+attribution: Limin Wang
+weight: 87
+keywords: [authorization, Role Based Access Control, security]
+---
+
+Micro-segmentation is a security technique that creates secure zones in cloud deployments and allows organizations to
```

Comment:
Let's add an entry to our glossary for this, since even Wikipedia doesn't define this term.